Lib/uuid.py

```diff
@@ -601,6 +632,22 @@ def _generate_time_safe():
                     _uuid_generate_time(_buffer)
                     return bytes(_buffer.raw), None
                 break
+        # if _uuid_generate_time has not been found (Linux) then we try libc
+        # as libc is already dynamically linked (or found above) verify a valid
+        # lib value, then look for uuid_generate()
+        if not lib:
+            try:
+                lib = ctypes.CDLL(None)
```

Comment:
How is this supposed to work? `None` arg effect is undocumented. According to https://www.programcreek.com/python/example/1108/ctypes.CDLL , this is going to get the main program itself (in POSIX, at least. In Windows, it raises a `TypeError`) which doesn't have the required export (and probably any). Looks redundant.